lib/mandrill_dm/delivery_method.rb

```diff
@@ -11,7 +11,9 @@ def deliver!(mail)
       message = Message.new(mail)
       @response = mandrill_api.messages.send(
         message.to_json,
-        MandrillDm.configuration.async
+        MandrillDm.configuration.async,
+        "Main Pool",
```

Comment:
`If you do not have any dedicated IPs, this parameter has no effect`
I think it's better to use `nil` - it won't confuse anyone.